src/hooks/v1/contractReader/UserHasPermission.ts

```diff
@@ -0,0 +1,32 @@
+import { NetworkContext } from 'contexts/networkContext'
+import { useContext } from 'react'
+import { V1ProjectContext } from 'contexts/v1/projectContext'
+import { V1OperatorPermission } from 'models/v1/permissions'
+
+import { useProjectOwner } from './ProjectOwner'
+import { useHasPermissions } from './HasPermissions'
+
+export function useUserHasPermission(
```

Comment:
i've been starting to lean towards using `wallet`/`connectedWallet` instead of `user` (could maybe do the same for `UserContext`, etc). thoughts on using `useConnectedWalletHasPermission` here? also happy to keep this as is for consistency and maybe change later

Reply:
Totally agree here. cc @jbx-protocol/frontend, note Peri's suggestion here ❤️